src/main/java/de/retest/recheck/ui/diff/IdentifyingAttributesDifferenceFinder.java

```diff
@@ -15,19 +15,25 @@
 
 	public IdentifyingAttributesDifference differenceFor( final IdentifyingAttributes expected,
 			final IdentifyingAttributes actual ) {
+		return differenceFor( expected, actual, GloballyIgnoredAttributes.getInstance() );
+	}
+
+	IdentifyingAttributesDifference differenceFor( final IdentifyingAttributes expected,
+			final IdentifyingAttributes actual, final GloballyIgnoredAttributes ignored ) {
 		Objects.requireNonNull( expected, "Expected cannot be null!" );
 		Objects.requireNonNull( actual, "Actual cannot be null!" );
 
 		final List<AttributeDifference> attributeDifferences = new ArrayList<>();
 
 		for ( final Attribute expectedAttr : expected.getAttributes() ) {
-			if ( Double.compare( expectedAttr.getWeight(), 0.0d ) == 0 || expectedAttr.isNotVisible() ) {
+			if ( Double.compare( expectedAttr.getWeight(), Attribute.IGNORE_WEIGHT ) == 0
```

Comment:
`if` should be removed, because weight is supposed to be only used for matching, not for differencing. (Adapt tests accordingly.)